test/views/PaymentAllocationViewSpec.scala

```diff
@@ -67,6 +67,7 @@ class PaymentAllocationViewSpec extends ViewSpec with ImplicitDateFormatter {
   val paymentAllocationsPoa1Nic4: String = messages("paymentAllocation.paymentAllocations.poa1.nic4")
   val paymentAllocationsPoa2Nic4: String = messages("paymentAllocation.paymentAllocations.poa2.nic4")
   val moneyOnAccountNA: String = s"${messages("paymentAllocation.moneyOnAccount")} ${messages("paymentAllocation.na")}"
+  val moneyOnAccountDueDate: String = s"${messages("paymentAllocation.moneyOnAccount")} 31 Jan 2021"
```

Comment:
you combining language independent resource with language fixes suffix;

Reply:
Changes Made